diff --git a/node_modules/react-native-vision-camera/android/src/main/java/com/mrousavy/camera/core/CameraSession.kt b/node_modules/react-native-vision-camera/android/src/main/java/com/mrousavy/camera/core/CameraSession.kt
index 862c03a..cbcd07b 100644
--- a/node_modules/react-native-vision-camera/android/src/main/java/com/mrousavy/camera/core/CameraSession.kt
+++ b/node_modules/react-native-vision-camera/android/src/main/java/com/mrousavy/camera/core/CameraSession.kt
@@ -90,7 +90,9 @@ class CameraSession(internal val context: Context, internal val callback: Callba
     }
   }

-  override fun getLifecycle(): Lifecycle = lifecycleRegistry
+  // override fun getLifecycle(): Lifecycle = lifecycleRegistry
+  override val lifecycle: Lifecycle
+    get() = lifecycleRegistry

   /**
    * Configures the [CameraSession] with new values in one batch.
